website/docs/debug.md

```diff
@@ -54,9 +54,11 @@ In debugging decisions and constraints, a few pieces of information can be helpf
    * A trace of the evaluation
    * The input document being evaluated
 
-Writing out this information for every request would be very expensive, and it would be hard
-to find the relevant logs for a given request. Instead, Gatekeeper allows users to specify
-resources and requesting users for which information will be logged. They can do so by
+Writing out all the information above for every request would be very expensive in terms of memory and load on the API server. It would also be hard
```

Comment:
FWIW it's the Gatekeeper server that would experience load and increased latency, which may lead to requests timing out, affecting the performance of the API server.